<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ErrorsWidget</class>
 <widget class="QWidget" name="ErrorsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>22</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="prevError">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>See previous error</string>
     </property>
     <property name="text">
      <string notr="true">&lt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="dismissCurrent">
     <property name="toolTip">
      <string>Dismiss error</string>
     </property>
     <property name="text">
      <string notr="true">x</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="nextError">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>See next error</string>
     </property>
     <property name="text">
      <string notr="true">&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="dismissAll">
     <property name="toolTip">
      <string>Dismiss all errors</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
